subkey/src/main.rs

```diff
@@ -18,15 +18,18 @@
 #[cfg(feature = "bench")]
 extern crate test;
 
-use std::{str::FromStr, io::{stdin, Read}};
+use std::{str::FromStr, io::{stdin, Read}, convert::TryInto};
 use hex_literal::hex;
 use clap::load_yaml;
 use bip39::{Mnemonic, Language, MnemonicType};
-use substrate_primitives::{ed25519, sr25519, hexdisplay::HexDisplay, Pair, Public, crypto::Ss58Codec, blake2_256};
+use substrate_primitives::{
+	ed25519, sr25519, hexdisplay::HexDisplay, Pair, Public,
+	crypto::{Ss58Codec, set_default_ss58_version, Ss58AddressFormat}, blake2_256
+};
 use parity_codec::{Encode, Decode, Compact};
 use sr_primitives::generic::Era;
 use node_primitives::{Balance, Index, Hash};
-use node_runtime::{Call, UncheckedExtrinsic, BalancesCall};
+use node_runtime::{Call, UncheckedExtrinsic, /*CheckNonce, TakeFees, */BalancesCall};
```

Comment:
Commented code

Reply:
yeah - it's there as a convenience measure - it'll be immediately uncommented once #3102 is in.